src/webui/webapplication.cpp

```diff
@@ -769,6 +771,33 @@ void WebApplication::action_command_bottomPrio()
     BitTorrent::Session::instance()->bottomTorrentsPriority(hashes);
 }
 
+void WebApplication::action_command_setLocation()
+{
+    CHECK_URI(0);
+    CHECK_PARAMETERS("hashes" << "location");
+
+    QStringList hashes = request().posts["hashes"].split("|");
+    QString newLocationPath = request().posts["location"].trimmed();
+
+    // check location exists
+    QFileInfo newLocation(newLocationPath);
+    if (!newLocation.exists()) {
+        qDebug() << "Location" << newLocationPath << "is invalid. Not moving files.";
+        return;
+    }
+
+    foreach (const QString &hash, hashes) {
+        BitTorrent::TorrentHandle *const torrent = BitTorrent::Session::instance()->findTorrent(hash);
+        if (torrent) {
+            // get old location
+            const QString oldLocation = torrent->savePath();
+            qDebug() << "Moving torrent" << torrent->hash() << "to" << newLocationPath;
```

Comment:
`TransferListWidget::setSelectedTorrentsLocation` [logs](https://github.com/qbittorrent/qBittorrent/blob/master/src/gui/transferlistwidget.cpp#L249) both the old location and the new location. Maybe the debug message should be reworded, but I do think it's valuable.